### ISSUE TITLE
hawck-inputd crashes if usb subsystem not present

### ISSUE THREAD

**Issue by lgbaldoni** [snyball/Hawck#47]
I know it's an edge case, but I was testing Hawck in a VM, and:
```
> hawck-inputd 
terminate called after throwing an instance of 'std::filesystem::__cxx11::filesystem_error'
  what():  filesystem error: directory iterator cannot open directory: No such file or directory [/dev/input/by-id]
Aborted (core dumped)
```

**Comment by snyball** [comment]
@lgbaldoni Yeah it is an edge case, but of course `inputd` should definitely recover from this situation. It's a very simple fix, so I'll be able to get it resolved shortly.

Thanks for the report.

**Comment by snyball** [comment]
PS: `hawck-inputd --no-hotplug --kbd-device <device>` can be used to work around this for the time being.

**Comment by snyball** [comment]
Fixed in f896bae22885fb6700f62b5633345bebe8b2a204
